packages/python/accern_xyme/accern_xyme.py

```diff
@@ -2921,26 +2925,30 @@ def upload_zip(
             for blob_uri in files
         ]
 
-    def upload_sklike_model_file(
+    def upload_model_file(
             self,
             model_obj: IO[bytes],
-            xcols: List[str],
+            xcols: Optional[List[str]],
+            ycols: Optional[List[str]],
             is_clf: bool,
             model_name: str,
-            maybe_classes: Optional[List[str]] = None,
-            maybe_range: Optional[
-                Tuple[Optional[float], Optional[float]]] = None,
+            version: int = -1,
+            model_params: Dict[str, Any] = None,
```

Comment:
`model_params` cannot be `None` as it is not Optional. Move model_params to above `version` and remove a default value. Same for the function below.

Reply:
Actually, let it be. Just change the type to be` Optional[Dict[str, Any]]`.